packages/dds/tree/src/feature-libraries/modular-schema/modularChangeTypes.ts

```diff
@@ -31,12 +34,36 @@ export interface ModularChangeset extends HasFieldChanges {
 	readonly revisions?: readonly RevisionInfo[];
 	readonly fieldChanges: FieldChangeMap;
 	readonly nodeChanges: ChangeAtomIdMap<NodeChangeset>;
+
+	// TODO: Should this be merged with `nodeChanges`?
+	readonly nodeToParent: ChangeAtomIdMap<FieldId>;
+	readonly nodeAliases: ChangeAtomIdMap<NodeId>;
```

Comment:
Which parts of ModularChangeset are expected to contain non-normalized `NodeId`s?

Reply:
Good question. I think the answer is that any node ID stored in the changeset could refer to an alias.

Reply:
Doesn't that mean all lookups in `nodeChanges` should be done by a function that tries to lookup each alias on the chain until it gets to the nomarlized ID?
For example the `encodeNode` function in `encodeFieldChangesForJson` doesn't do any normalization.

Reply:
I realized we should only have non-normalized node IDs in the field IDs in the values of `nodeToParent` and `crossFieldKeys`. I've added some comments documenting this.